gammapy/image/maps.py

```diff
@@ -260,8 +260,8 @@ def solid_angle(self):
         """
         Solid angle image
         """
-        xsky, ysky = self.coordinates(mode='edges')
```

Comment:
@cdeil 
Get an error:
```
ValueError: too many values to unpack
```
with the old version. You changed the coordinates method of the skymap that return a tuple if coord_type=galactic but an array if coord_type=skycoord.
Is it ok what I did?